test/Parse/availability_query_unavailability.swift

```diff
@@ -80,9 +80,6 @@ if #unavailable(OSX 10.51, iOS 8.0) {
 if #unavailable(OSX 10.51, { // expected-error {{expected platform name}} // expected-error {{expected ')'}} expected-note {{to match this opening '('}}
 }
 
-if #unavailable(OSX 10.51,) { // expected-error {{expected platform name}}
-}
-
```

Comment:
Would it make sense to keep these deleted tests around to make sure that we now parse them correctly?

Reply:
I delete here because I added a similar test case to trailing comma test files.

Reply:
Ah, I missed that. Thanks.